pegasus/sites.v3/hourofcode.com/views/mobilecheck.js.haml

```diff
@@ -0,0 +1,13 @@
+:javascript
```

Comment:
Instead of having this in a haml file, couldn't we put it in our JS and not live on window? i.e. have TutorialExplorerManager do our mobilecheck for us?